### PR TITLE
Update the content-strategy.md and mention the fixed docu order

### PR DESCRIPTION
**Pull request by klaudiagrz**
**Description**

Changes proposed in this pull request:

- Update the content-strategy.md and mention the fixed docu types order
- Update the order of the templates in the table
- Add "backend" and "frontend" to the guidelines
- Update the gsg template and turn it into the tutorial template

### CODE REVIEW

File: guidelines/content-guidelines/content-strategy.md [hunk 1]
```diff
@@ -43,17 +43,18 @@ There is a set of documents that a given technical topic must include. You can a
 
 ### Obligatory
 
-Each technical topic must have the following document types:
+Each technical topic must have the following document types arranged in the fixed order listed below:
```
Comment (kazydek) [inline guidelines/content-guidelines/content-strategy.md]:
```suggestion
Each technical topic must have the following document types arranged in the following order:
```

**Comment by kazydek** [comment]
Pls also update the GS template into Tutorial.

**Comment by klaudiagrz** [comment]
> Pls also update the GS template into Tutorial.

Yup, already done that.

File: guidelines/content-guidelines/content-strategy.md [hunk 2]
```diff
@@ -62,7 +63,6 @@ Each technical topic must have the following document types:
 ### Optional
 
 You can add the following document type to the Kyma documentation:
```
Comment (kazydek) [inline guidelines/content-guidelines/content-strategy.md]: And  maybe we should add info here that these should go after the obligatory types?

**Pull request merged**